assets/dashboard.js

```diff
@@ -146,6 +148,13 @@ const Filtering = class {
         visible = false;
       if (visible && this.only_category && this.only_category != category)
         visible = false;
+      if (visible && this.only_search.trim() !== '') {
+        const nameAttr = elems[i].getAttribute('data-name').toLowerCase();
+        const descAttr = elems[i].getAttribute('data-desc') ? elems[i].getAttribute('data-desc').toLowerCase() : '';
```

Comment:
Can you retrieve the description from the DOM below instead? This would force duplicating the descriptions